src/main/scala/com/campudus/tableaux/database/domain/file.scala

```diff
@@ -0,0 +1,66 @@
+package com.campudus.tableaux.database.domain
+
+import java.util.UUID
+
+import com.campudus.tableaux.database.model.FolderModel.FolderId
+import org.joda.time.DateTime
+import org.vertx.scala.core.json._
+
+object File {
+  def apply(name: String, mimeType: String, description: String): File = {
+    File(None, name, description, mimeType, name, None, None, None)
+  }
+
+  def apply(uuid: UUID, name: String, mimeType: String, description: String): File = {
+    File(Option(uuid), name, description, mimeType, name, None, None, None)
+  }
+
+  def apply(uuid: UUID, name: String, mimeType: String): File = {
+    File(Option(uuid), name, null, mimeType, name, None, None, None)
+  }
+}
+
+case class File(uuid: Option[UUID],
```

Comment:
Is "Attachment" a naming alternative? As there is `File` in the standard library, the current name might lead to confusion. WDYT?